src/features/Home/screens/HomeScreen/index.tsx

```diff
@@ -1,12 +1,89 @@
-import React, {FC} from 'react';
+import React, {FC, useCallback, useEffect, useState} from 'react';
+import {FlatList, ListRenderItem, TouchableOpacity, View} from 'react-native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {useMMKVString} from 'react-native-mmkv';
 import {SafeAreaView} from 'react-native-safe-area-context';
-import TextWrapper from '../../../../components/TextWrapper';
 import {styles} from './styles';
+import Tweet, {ITweet} from '../../components/Tweet';
+import Loader from '../../../../components/Loader';
+import TextWrapper from '../../../../components/TextWrapper';
+import TweetsAPI from '../../../../services/API/Tweets';
+import {HomeStackParamList} from '../../../../models/navigation';
+
+const ListEmptyComponent: FC = () => {
+  return <TextWrapper>No items</TextWrapper>;
+};
+
+const keyExtractor = (item: ITweet) => {
+  return item.id;
+};
+
+const ItemSeparatorComponent: FC = () => {
+  return <View style={styles.separator} />;
+};
+
+type Props = NativeStackScreenProps<HomeStackParamList, 'Home'>;
+
+const HomeScreen: FC<Props> = ({navigation}) => {
+  const [userName] = useMMKVString('nickname');
```

Comment:
'nickname' is a magic string. Ideally you should have a enum with all possible values for mmkv